src/Filament/Resources/TenantResource/Pages/CreateTenant.php

```
@@ -110,12 +114,18 @@ public function create(bool $another = false): void
     {
-        \Log::info("Saving Tenant");
+        Log::info("Saving Tenant");
         $record = new Tenant(collect($data)->except('domain')->toArray());
         $record->saveOrFail();
-        \Log::info("Saving Domains");
+        Log::info("Saving Domains");
         $record = $record::find($record->id);
         $record->domains()->create(['domain' => collect($data)->get('domain')]);
+        Artisan::call('route:clear');
         return $record;
     }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'createTenantRecord'. (Unused Code Rules)

(UnusedPrivateMethod)